vcd/resource_vcd_org_vdc.go

```diff
@@ -249,7 +259,14 @@ func resourceVcdOrgVdc() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Computed:    true,
-				Description: "ID of default VM sizing policy ID",
+				Deprecated:  "Use `default_compute_policy_id` attribute instead, which can support VM Sizing Policies, VM Placement Policies and vGPU Policies",
+				Description: "ID of default VM Compute policy, which can be a VM Sizing Policy, VM Placement Policy or vGPU Policy",
+			},
+			"default_compute_policy_id": {
```

Comment:
Should we use `ExactlyOneOf` for `default_vm_sizing_policy_id` and `default_compute_policy_id` to prevent users populating both?

Reply:
That's a good idea 🙂

Reply:
Added!

Reply:
Suggested wrong validation field. `ExactlyOneOf` was the right one.